### PR TITLE
add tests for daily feature

### PR DESCRIPTION
**Pull request by yingqi0607**

### CODE REVIEW

**Comment by cwenling** [comment]
@codecov-io

please comment

File: src/test/java/seedu/tr4cker/logic/commands/DailyCommandTest.java [hunk 1]
```diff
@@ -0,0 +1,87 @@
+package seedu.tr4cker.logic.commands;
+
+
```
Comment (ruilingk) [inline src/test/java/seedu/tr4cker/logic/commands/DailyCommandTest.java]: extra line

File: src/test/java/seedu/tr4cker/logic/parser/DailyCommandParserTest.java [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+import static seedu.tr4cker.logic.parser.CommandParserTestUtil.assertParseFailure;
+import static seedu.tr4cker.logic.parser.CommandParserTestUtil.assertParseSuccess;
+import static seedu.tr4cker.logic.parser.ParserUtil.MESSAGE_INVALID_INDEX;
+
+import org.junit.jupiter.api.Test;
+
+import seedu.tr4cker.commons.core.index.Index;
+import seedu.tr4cker.logic.commands.DailyCommand;
+
+public class DailyCommandParserTest {
```
Comment (ruilingk) [inline src/test/java/seedu/tr4cker/logic/parser/DailyCommandParserTest.java:18]: missing javadocs

File: src/test/java/seedu/tr4cker/logic/parser/TodoCommandParserTest.java [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+import static seedu.tr4cker.logic.parser.CommandParserTestUtil.assertParseFailure;
+import static seedu.tr4cker.logic.parser.CommandParserTestUtil.assertParseSuccess;
+import static seedu.tr4cker.logic.parser.ParserUtil.MESSAGE_INVALID_INDEX;
+import static seedu.tr4cker.testutil.TypicalIndexes.INDEX_FIRST_TASK;
+
+import org.junit.jupiter.api.Test;
+
+import seedu.tr4cker.logic.commands.TodoCommand;
+
+public class TodoCommandParserTest {
```
Comment (ruilingk) [inline src/test/java/seedu/tr4cker/logic/parser/TodoCommandParserTest.java:15]: missing javadocs

File: src/test/java/seedu/tr4cker/model/daily/TodoTest.java [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+
+import java.util.Objects;
+
+import org.junit.jupiter.api.Test;
+
+import seedu.tr4cker.model.task.Deadline;
+import seedu.tr4cker.model.task.Name;
+
+
+public class TodoTest {
```
Comment (ruilingk) [inline src/test/java/seedu/tr4cker/model/daily/TodoTest.java:20]: here too hehe and the bottom 3 classes too

**Review by ruilingk — COMMENTED** [review]
minor tweaks and it lgtm

**Review by ethan-l-m-e — COMMENTED** [review]
Nice naming for test case method names.

**Review by ethan-l-m-e — APPROVED** [review]
lgtm

**Pull request merged**